site/docs/v1/tech/installation-guide/docker.adoc

```diff
@@ -178,6 +178,19 @@ For example if startup is failing and you see the following in the logs, you wil
 2018-11-22T13:00:05.346606495Z [2]: max virtual memory areas vm.max_map_count [65530] is too low, increase to at least [262144]
 ----
 
+== Upgrading
+
+To upgrade FusionAuth when running with `docker-compose`:
+
+. Stop the instance: `docker-compose down`.
+. Modify the `docker-compose.yml` file (or the `docker-compose.override.yml` file, if applicable) to point to the version of FusionAuth you want. You can see https://hub.docker.com/r/fusionauth/fusionauth-app[available tags].
+. Start it up: `docker-compose up`.
+. Login to the administrative UI. If there were migrations required, they'll be applied automatically to your database.
```

Comment:
This will depend upon your runtime mode. 
https://fusionauth.io/docs/v1/tech/installation-guide/fusionauth-app#runtime-modes

Reply:
Ah, great. I'll note that it depends on the mode.